src/openforms/contrib/objects_api/api/views.py

```diff
@@ -88,7 +88,7 @@ def get_objects(self) -> list[dict[str, Any]]:
         config_group = input_serializer.validated_data["objects_api_group"]
 
         with get_objecttypes_client(config_group) as client:
-            return client.list_objecttype_versions(self.kwargs["submission_uuid"])
+            return client.list_objecttype_versions(self.kwargs["objecttype_uuid"])
 
 
 class TargetPathsListView(views.APIView):
```

Comment:
so to address the comment on the API schema file:

this view belongs in `openforms.registrations.contrib.objects_api.api.views`, since it's specific to the registration plugin - we can't re-use it for the prefill plugin.